omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -125,7 +134,48 @@ public String post(PageModel model, @MethodParam("getAccount") @BindParams Accou
 			
 			String forcePassword = otherAccountData.getForceChangePassword() ? "true" : "false";
 			user.setUserProperty(OpenmrsConstants.USER_PROPERTY_CHANGE_PASSWORD, forcePassword);
-			
+
+            List<Extension> customUserFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountUserFragments");
+            Map<String, String[]> parameterMap = request.getParameterMap();
+            for(Extension ext : customUserFragments) {
+                if (StringUtils.equals(ext.getExtensionParams().get("type").toString(), "userProperty")) {
+                    String userPropertyName = ext.getExtensionParams().get("userPropertyName").toString();
+                    String[] parameterValues = parameterMap.get(userPropertyName);
+                    if (parameterValues != null && parameterValues.length > 0) {
+                        if (parameterValues.length > 1) {
+                            log.warn("Multiple userProperty for a single user type not supported, ignoring extra values");
+                        }
+                        String parameterValue = parameterValues[0];
+                        if (userPropertyName !=null && parameterValue != null) {
+                            user.setUserProperty(userPropertyName, parameterValue);
+                        }
+                    }
+                }
+            }
+
```

Comment:
Can we do some bit of uniformity with our indention?

Reply:
Sorry, I didn't get you properly

Reply:
The indention is not the same. Can you look at it again?

Reply:
Do you mention the implementation after this line which points to person attribute? I used the existing patient registration implementation to build this one. Any brief comments to make this better?

Reply:
Can you look at lines 138 and 156?

Reply:
At what vertical column numbers do they start? In other words, do they have the same indention?

Reply:
Just got it, and fixed it as well. Please see the updated code